mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -176,6 +178,7 @@ MMCH:
 		TurnSpeed: 20
 		Speed: 56
 		AlwaysTurnInPlace: true
+		TerrainOrientationAdjustmentMargin: -1
```

Comment:
`JUGG` needs this, too!

Reply:
Unless this is only needed by actors with voxel elements, but then `SMECH` wouldn't need it, either.